src/input_xml.F90

```diff
@@ -1049,8 +1049,9 @@ subroutine read_geometry_xml()
     do i = 1, n_cells
       c => cells(i)
 
-      ! Initialize the number of cell instances - this is a base case for distribcells
+      ! Initialize distribcell instances and distribcell index
```

Comment:
Ok I see. Sorry for forgetting what these things are - the (old and new) naming convention even confuses me!